PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -200,6 +202,13 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     resolution = JME::JetResolution::get(setup, m_jets_algo_pt);
                     resolution_sf = JME::JetResolutionScaleFactor::get(setup, m_jets_algo);
                 }
+
+                if(m_useDeterministicSeed) {
+                    unsigned int runNum_uint = static_cast <unsigned int> (event.id().run());
+                    unsigned int evNum_uint = static_cast <unsigned int> (event.id().event()); 
+                    std::uint32_t seed = runNum_uint + 4*evNum_uint;
+                    m_random_generator.seed(seed);
```

Comment:
should a lumisection be a part of this formula?
MC occasionally starts from the same but has a different LS
.. although there are MC gen cases where events repeat as well.

I'd add the first jet pt or something like that to randomize the result and avoid repeated values in factually different events.

Reply:
@slava77 what is your recommendation?

Reply:
One option is to repeat the random seed for the searing factors (applied different jets) every run. I wouldn't think this can introduce any unwanted bias and harm.
Making the seed depend on jet pt, could make the existing differences between jets made from AOD and MiniAOD slightly bigger. Also this isn't a serious harm.
@rappoccio What's your take?